README.md

```diff
@@ -1,55 +1,110 @@
-# Prison Visit Booking email
+# Visit someone in prison
 
 [![Build Status](https://travis-ci.org/ministryofjustice/prison-visits.png?branch=master)](https://travis-ci.org/ministryofjustice/prison-visits)
 [![Code Climate](https://codeclimate.com/github/ministryofjustice/prison-visits.png)](https://codeclimate.com/github/ministryofjustice/prison-visits)
 [![Code Coverage](https://codeclimate.com/github/ministryofjustice/prison-visits/coverage.png)](https://codeclimate.com/github/ministryofjustice/prison-visits)
 
-Also known as "PVBE", is the Alpha system for the Prison Visit Booking exemplar. 
+Also known as Prison Visits Booking, is one of the 25 [GOV.UK](https://www.gov.uk/) 'exemplar' [digital transformation services](https://www.gov.uk/transformation).
```

Comment:
This might be better written as:

> Also known as “Prison visit booking”, is one of the 25 [‘exemplar’ government digital transformation services](https://www.gov.uk/transformation).

That way, the “also known as” refers to the same name as on the transformation page, and the transformation link better describes what it is (transformation is a GDS thing, not a GOV.UK thing).